src/site/layouts/step_by_step.drupal.liquid

```diff
@@ -0,0 +1,71 @@
+{% include "src/site/includes/header.html" with drupalTags = true %}
+
+{% include "src/site/includes/alerts.drupal.liquid" %}
+{% include "src/site/includes/preview-edit.drupal.liquid" %}
+{% include "src/site/includes/breadcrumbs.drupal.liquid" %}
+
+<div id="content" class="interior">
+  <main class="va-l-detail-page">
+    <div class="usa-grid usa-grid-full">
+      <div class="usa-width-three-fourths">
+        {% if !entityPublished %}
+          <div class="usa-alert usa-alert-info">
+            <div class="usa-alert-body">
+              <p class="usa-alert-text">You are viewing a draft.</p>
+            </div>
+          </div>
+        {% endif %}
+
+        <article class="usa-content">
+          <!-- Title -->
+          <h1>{{ title }}</h1>
+
+          <!-- Intro -->
+          <p>{{ fieldIntroTextLimitedHtml.processed }}</p>
+
+          <!-- Alert -->
+          {% if fieldAlertSingle %}
+            {% include "src/site/paragraphs/alert_single.drupal.liquid" with entity = fieldAlertSingle.entity %}
+          {% endif %}
+
+          <!-- Buttons -->
+          {% for fieldButton in fieldButtons  %}
+            {% include "src/site/paragraphs/button.drupal.liquid" with entity = fieldButton.entity %}
+          {% endfor %}
```

Comment:
There might be a `fieldButtonsRepeat` property to indicate if the buttons should render on both the top and bottom of the page. I'm not sure which ones it drives though.

For example, https://github.com/department-of-veterans-affairs/vets-website/pull/14178/files#diff-3c70fd41c419fb9398a236d62972f316R29

Reply:
Oh interesting, but we would only want that conditional on the second set of buttons at the bottom of the page right? I feel like you'd always want to at least show them at the top of the page?

Reply:
IDK because the other argument might be that we want them to read the page before prompting them to do something

Reply:
I'll ask!

Reply:
Thanks! Let me know what they say! 🙂 https://dsva.slack.com/archives/C0173CAJ7A6/p1599844808147300